### PR TITLE
Added getter and setter functions for ROI

### PR DESCRIPTION
**Pull request by TheIllusionist77**
I was not able to find getter and setter functions for changing the ROI of the sensor, so I attempted to create my own.

This is my first time creating a pull request, so I apologize in advance if I messed something up.

### CODE REVIEW

**Comment by FoamyGuy** [comment]
@TheIllusionist77 Thanks for working on this. 

The easiest way to get the issues with the actions worked out is to install and run pre-commit and then commit / push the resulting changes, it'll fix up the code format and things for you.  This page details the process: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code

For simple getters and setters we prefer to have properties instead of just getter / setter functions. https://docs.circuitpython.org/en/latest/docs/design_guide.html#getters-setters It's easy to convert functions to properties with decorators see an example in the timing_budget properties within this library: https://github.com/adafruit/Adafruit_CircuitPython_VL53L1X/blob/main/adafruit_vl53l1x.py#L235-L244

**Comment by TheIllusionist77** [comment]
Definitely many more commits than needed - it's my first time using pre-commit and actually the Git command line in general.

Anyway, is this good for the ROI getters and setters?

File: adafruit_vl53l1x.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 from adafruit_bus_device import i2c_device
 from micropython import const
 
-__version__ = "0.0.0+auto.0"
+__version__ = "1.1.14"
```
Comment (FoamyGuy) [inline adafruit_vl53l1x.py]: Please change the version back to `0.0.0+auto.0` the real values get filled in automatically during the release process.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]

**Review by FoamyGuy — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**